src/github.com/matrix-org/rageshake/submit.go

```diff
@@ -38,6 +39,11 @@ type submitServer struct {
 	// github client for reporting bugs. may be nil, in which case,
 	// reporting is disabled.
 	ghClient *github.Client
+
+	APIPrefix string
+
+	// mappings from application to github owner/project
+	GithubProjectMappings map[string]string
```

Comment:
There's no real distinction here between public and private given it's all in the same package and isn't used outside of it (fun fact: it *can't* be used outside because you cannot export `package main`). I'd keep it all private.

Also, one-liner on API prefix would be schweet please.